src/main/java/com/movie_collection/be/Category.java

```diff
@@ -0,0 +1,6 @@
+package com.movie_collection.be;
+
+import javafx.beans.property.StringProperty;
+
+public record Category(int id, StringProperty name) {
```

Comment:
we dont have to use IntegerProperty for this as well to map it or am I wrong ?

Reply:
id will probably never be shown to the user so binding it to the table won't be necessary
I would stick with primitive

Also category was removed from this PR but the same discussion is for movie class

Reply:
good argument ! I agree